mlflow/ml-package-versions.yml

```diff
@@ -671,6 +671,8 @@ langchain:
           "langchain-core!=0.1.39", # https://github.com/langchain-ai/langchain/issues/19947
           "langchain-openai",
           "langgraph",
+          "fastapi",
+          "uvicorn",
```

Comment:
`tests/openai/mock_openai.py` requires `fastapi` and `uvicorn`.

Reply:
The `llama_index` flavor also uses `mock_openai` fixture. Could you add these dependencies there too?